### PR TITLE
Unit-2.2-CSS: Basic Styling

### PR DESCRIPTION
**Pull request by nathanjophillips**
## The Purpose
- Gain a basic understanding of CSS
- Apply styles to the project to make things visually appealing
- Use CSS to change the layout of the project in a meaningful, UX-improving way

#### 1. Your CSS
 - [x]  is housed in a single CSS file
 - [x] is utilized on every page
 - [x] uses at least one class selector to standardize styles for a type of HTML object.
 - [x] defines a standard layout for each page to follow.

#### This adds style the pages: 
- User Profile
- User List
- Create User
- Edit User 

### The changes to look for are:
1. The Nav bar is uniform on each page
2. Card sizes and spacing are user friendly
3. Fonts are all the same
4. Drop shadow and color are the same on each page
5. No overlapping

## How To Test
- Clone Repo [nathan-phillips-ux-iop](https://github.com/phillnab/nathan-phillips-ux-iop)
- Checkout branch `nathan-phillips-ux-iop/unit-2.2-css`
- Open each `html file` to view locally
- All styling will be in the `CSS` directory
- Check to see if your pages look like the following screenshots

#### Check Styling of:
- Nav bar
- Card spacing and sizing
- Dropshadow
- No overlapping 

screen shot 2019-02-26 at 10 45 26 am

## User List:
 Check Styling of: **
- Nav bar
- Card spacing and sizing
- Dropshadow
- No overlapping 

screen shot 2019-02-26 at 10 45 34 am

##### Check Styling of: 
- Nav bar
- Card spacing and sizing
- Dropshadow
- No overlapping 

screen shot 2019-02-26 at 10 45 41 am
##### Check Styling of:
- Nav bar
- Card spacing and sizing
- Dropshadow
- No overlapping 

screen shot 2019-02-26 at 10 45 49 am

## Checklist

Under penalty of public shaming, I avow that I:

- [x] Reviewed the diff to look for typos, whitespace errors, and console.log() statements
- [ ] Verified that there are no compiler or linter errors or warnings
- [ ] Verified the build in production(optimized) mode
- [x] Updated the docs, if necessary
- [x] Included the appropriate labels
- [ ] Referenced applicable issues (i.e. Closes #XXXX, Fixes #XXXX)

Browsers tested:
- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] Internet Explorer

### CODE REVIEW

File: html/edit-user.html [hunk 1]
```diff
@@ -32,8 +34,8 @@
       <div>    
         <label for="email">Email:</label>
         <input type="text" name="email" id="email">
-      </div>    
-      <button type="submit">
+      </div>
+      <button type="submit">Finish Edit</button>
```
Comment (Jropp) [inline html/edit-user.html]: 'Save' or 'Save Edit' might be a little more intuitive here.

File: html/user-list.html [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <head>
     <title>User List</title>
     <meta content="text/html; charset=UTF-8">
+    <link rel="stylesheet" type="text/css" href="../css/style.css" />
 </head>
 <body>
```
Comment (Jropp) [inline html/user-list.html:8]: The user cards don't have fixed width, so the "Jordan Smith" card is slightly wider than the others.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
```
Comment (Jropp) [inline css/style.css:1]: You might run into some fun issues with importing a font through a URL once `webpack` is added to the project. If you aren't married to the Poppins (he he) font, something built into the browser might save you some time and energy.

**Review by Jropp — CHANGES_REQUESTED** [review]
Overall it looks great, just a few small things.

File: css/style.css (hunk 3 above)
Comment (nathanjophillips) [reply css/style.css:1]: Yeah, I was thinking I might run into an issue with that.  I can change it pretty quick if need be.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+html, body {
+    font-family: "Poppins", sans-serif;
+    text-align: -webkit-center;
+    margin-top: 0px;
+}
```
Comment (Jropp) [inline css/style.css]: Looks great, just one more little thing from my end. Could you put some spacing in here between css

```
a {
  background-color: emoBlack
}
       <--- space here
ul {
  width: yourMom
}
```

**Review by Jropp — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 4 above)
Comment (nathanjophillips) [reply css/style.css]: no problem! haha

**Review by kddrake — APPROVED** [review]
@Jropp feel free to merge once you approve

**Review by Jropp — APPROVED** [review]

**Pull request merged**